tests/helpers/test_update_coordinator.py

```diff
@@ -450,6 +450,20 @@ async def test_coordinator_entity(
     assert len(crd._listeners) == 0
 
 
+async def test_schedule_refresh_point_in_utc_time(
+    hass: HomeAssistant,
+    crd: update_coordinator.DataUpdateCoordinator[int],
+) -> None:
+    """Test that subclasses overriding _schedule_refresh with async_track_point_in_utc_time continue to work."""
+    crd._handle_refresh_interval = AsyncMock()
+    event.async_track_point_in_utc_time(crd.hass, crd._job, utcnow())
+
+    # wait until the handler is called
+    await asyncio.sleep(0)
```

Comment:
This shouldn't be needed since block till done does it

Reply:
I thought so too but test fails without it. Perhaps this is the reason: https://github.com/home-assistant/core/pull/46861/files#r595173329